ethers-signers/src/trezor/app.rs

```diff
@@ -56,9 +61,41 @@ impl TrezorEthereum {
         Ok(())
     }
 
+    fn get_cached_session() -> Result<Option<Vec<u8>>, TrezorError> {
```

Comment:
is the session only a 32byte identifier?
need definitely some docs on this, or links to trezor api

Reply:
> is the session only a 32byte identifier?

yes. copied the relevant paragraphs into the PR, and added a reference on the code.